docs/howtos/resource-override.md

```diff
@@ -149,12 +162,44 @@ with the image `nginx:1.20.0` on all clusters with the label `env: prod`.
 >   ...
 >```
 
+To delete the `my-deployment` on the clusters with the label `env: test` selected by the clusterResourcePlacement `crp-example`,
+you can use the `Delete` override type.
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ResourceOverride
+metadata:
+  name: example-ro
+  namespace: test-namespace
+spec:
+  placement:
+    name: crp-example
+  resourceSelectors:
+    -  group: apps
+       kind: Deployment
+       version: v1
+       name: my-deployment
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: test
+        overrideType: Delete
+```
+

```

Comment:
why not move this example into the Override Type section?

Reply:
or move the OverrideType section earlier or remove it?   Ideally, the OverrideType section should be the parent of JSON Patch Override and have a delete type section. Currently, it's at the same level as JSON Patch and no corresponding Delete section.

Reply:
i think the current logic is to give concrete examples at the very beginning of the policy section, and then go deep into each fields. it should be ok?